src/Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core/Models/DevCanvasRequestV1.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif;
+
+using Newtonsoft.Json;
+
+namespace Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core.Models
+{
+    /// <summary>
+    /// Class representing the new format for how the DevCanvas endpoint is queried.
```

Comment:
```suggestion
    /// Class representing the format for how the DevCanvas endpoint is queried.
```

Nothing stays 'new' :)